k8s/base/media/kustomization.yaml

```
@@ -19,12 +19,13 @@ helmCharts:
     valuesInline:
       # cache jellyfin in worker node specific directory
       persistence:
+        config:
+          enabled: true
+          size: 1Gi
         cache:
           enabled: true
           type: emptyDir
           size: 2Gi
```

```suggestion
          size: 10Gi
```
Increase cache size to allow more downloads to proceed at once.